.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,29 @@
+name: Upload Python Package
```

Comment:
I added this to simplify pypi uploads - I use the same workflow in my projects

All you need to do is create a release and the rest will be taken care of by pypi

All you need to do is set up trusted publishing in your pypi settings (not hard to do)

https://docs.pypi.org/trusted-publishers/